framework/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    namespace 'org.eu.droid_ng.wellbeing.framework'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "org.eu.droid_ng.wellbeing.framework"
+        minSdk 29
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
+}
+
+dependencies {
+    //noinspection GradleDependency
```

Comment:
androidx.annotation is not really needed, it's just for @NonNull or stuff

Reply:
I did the first thing that came to mind, I think I should have removed it.